mlflow/utils/environment.py

```diff
@@ -18,6 +25,91 @@
 _CONDA_ENV_FILE_NAME = "conda.yaml"
 _REQUIREMENTS_FILE_NAME = "requirements.txt"
 _CONSTRAINTS_FILE_NAME = "constraints.txt"
+_PYTHON_ENV_FILE_NAME = "python-env.yaml"
+
+
+class EnvManager(Enum):
+    LOCAL = "local"
+    CONDA = "conda"
+    VIRTUALENV = "virtualenv"
+
+
+_PYTHON_REQUIREMENT_REGEX = re.compile(r"^python=([\d.]+)$")
```

Comment:
```suggestion
_PYTHON_REQUIREMENT_REGEX = re.compile(r"^python(=|==)([\d.]+)")
```

Reply:
conda's version specification is different from PEP: https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications

Reply:
Could you add a comment explaining what the regex match ?

Reply:
and `python(=|==)([\d.]+)` only handle "equal" case, what if user set "python<xxx" or "python>xxx" ? Ignore it ?